ParkenDD/Colors.swift

```diff
@@ -22,20 +22,34 @@ struct Colors {
 	*/
 	static func colorBasedOnPercentage(_ percentage: Double, emptyLots: Int) -> UIColor {
 
-		let hue = 1 - (percentage * 0.3 + 0.7) // I want to limit this to the colors between 0 and 0.3
+        let lotDensityColor: UIColor = {
```

Comment:
Good job. I hadn't thought about doing this before. I don't like you setting the constant like this but I also don't dislike it 🤔 

I want you to move it elsewhere but this is where it makes the most sense, so it means you've shown me a new way to pack code! Thanks!

Reply:
🤷‍♂️ I got that style from a YouTuber I watch.  I agree...it seems a little wonky but it also makes some sense.